.github/workflows/bump-auxiliary-packages.yml

```diff
@@ -36,11 +33,12 @@ jobs:
 
       - name: Install Dependencies
         run: |
-          npm ci
+          npm ci --workspace @mongosh/build
```

Comment:
this is just to help speed up the CI a little

Reply:
I'm a bit worried that this would lead to inconsistent behavior between local an CI usage (e.g. because some dependency might have optional dependencies that are present locally but absent in this task) but it's no big deal either way

Reply:
I feel like that'd actually be a good thing because it'd help force us to be more intentional with defining our dependencies but yeah if it causes some unexpected behavior, that could be annoying